pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -332,6 +334,10 @@ public PulsarService(ServiceConfiguration config,
 
         this.ioEventLoopGroup = EventLoopUtil.newEventLoopGroup(config.getNumIOThreads(), config.isEnableBusyWait(),
                 new DefaultThreadFactory("pulsar-io"));
+        // A shared io event group for broker clients to avoid internal clients from occupying IO
+        // threads for a long time and affecting the throughput of the broker.
+        this.brokerClientSharedIoEventLoopGroup = EventLoopUtil.newEventLoopGroup(config.getNumIOThreads(),
```

Comment:
`config.getNumIOThreads()` 
If we consider using the different values for the internal clients, we can introduce the new config in the future.